fe/fe-core/src/main/java/com/starrocks/privilege/PrivilegeCollection.java

```diff
@@ -272,7 +272,7 @@ public void removeInvalidObject(GlobalStateMgr globalStateMgr) {
                 PrivilegeEntry entry = entryIterator.next();
                 if (entry.object != null && !entry.object.isFuzzyMatching() && !entry.object.validate(globalStateMgr)) {
                     String entryStr = GsonUtils.GSON.toJson(entry);
-                    LOG.info("find invalidate object, will remove the entry now: {}", entryStr);
+                    LOG.info("find invalid object, will remove the entry now: {}", entryStr);
```

Comment:
why info level log？？

Reply:
Considering the situation we are removing a privilege entry object, it's necessary.

Reply:
It's not a common log.